NorthstarDedicatedTest/voicebasic.cpp

```diff
@@ -0,0 +1,37 @@
+#include "pch.h"
+#include "voicebasic.h"
+
+void BasicVoice::speakText(char* input)
+{
+	size_t size = strlen(input) + 1;
+	wchar_t* wtext = new wchar_t[size];
+
+	size_t outSize;
+	mbstowcs_s(&outSize, wtext, size, input, size - 1);
+	LPWSTR ptr = wtext;
+	hr = pVoice->Speak(ptr, SVSFlagsAsync + SVSFPurgeBeforeSpeak, NULL);
+}
+
+void BasicVoice::setRate(float rate)
+{
+	speechRate = rate;
+	pVoice->SetRate(rate);
+}
+void BasicVoice::setVolume(float volume)
+{
+	speechVolume = volume;
+	pVoice->SetVolume(volume * 100.0);
+}
+
+void BasicVoice::outSpeech()
+{
+	pVoice->Release();
+	pVoice = NULL;
+	::CoUninitialize();
+}
```

Comment:
What is the point of this method? It's not called anywhere, and it looks like it should be the destructor.

Reply:
It's a remnant of code that did call it, will sort that out